packages/infra/lib/api-stack.ts

```diff
@@ -379,6 +379,15 @@ export class APIStack extends Stack {
       });
     }
 
+    const fhirToBundleLambda = this.setupFhirBundleLambda({
+      lambdaLayers,
+      vpc: this.vpc,
+      fhirServerUrl: props.config.fhirServerUrl,
+      envType: props.config.environmentType,
+      sentryDsn: props.config.lambdasSentryDSN,
+      alarmAction: slackNotification?.alarmAction,
+    });
```

Comment:
We should avoiding adding even a single resource to the APIStack - its close to the limit of 500 resource. We hit this limit before and it was painful to fix - even worse, we had some stuff we could move, next time it will be much harder. 😓 

Can we move this to the LambdasNestedStack, please?